config/routes/api.rb

```diff
@@ -7,6 +7,8 @@
 
     get 'env_settings', to: 'env_settings#index'
 
+    patch 'locale', to: 'locales#update'
```

Comment:
we got session, profile, ... then it should also be called locale, not locales I think